crates/cli/src/subcommands/sql.rs

```diff
@@ -27,39 +28,31 @@ pub fn cli() -> clap::Command {
                 .conflicts_with("interactive")
                 .help("The SQL query to execute"),
         )
-        .arg(Arg::new("interactive")
-                 .long("interactive")
-                 .action(ArgAction::SetTrue)
-                 .conflicts_with("query")
-                 .help("Runs an interactive command prompt for `SQL` expressions"),)
+        .arg(
+            Arg::new("interactive")
+                .long("interactive")
+                .action(ArgAction::SetTrue)
+                .conflicts_with("query")
+                .help("Runs an interactive command prompt for `SQL` expressions"),
+        )
         .group(
             ArgGroup::new("mode")
-                .args(["interactive","query"])
+                .args(["interactive", "query"])
                 .multiple(false)
-                .required(true)
-        )
-        .arg(
-            Arg::new("identity")
-                .long("identity")
-                .short('i')
-                .conflicts_with("anon_identity")
-                .help("The identity to use for querying the database")
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.